handsontable/src/dataMap/metaManager/metaSchema.js

```diff
@@ -3237,6 +3237,22 @@ export default () => {
      */
     language: 'en-US',
 
+    /**
+     * Locale used for actions performed on data (for example, filtering, searching, comparing data).
```

Comment:
Do you think it's worth mentioning that the supported value must be valid [BCP47](https://www.rfc-editor.org/rfc/bcp/bcp47.txt) lang code format?

We could mention `locale` in `docs/guides/internationalization/internationalization-i18n.md`

Reply:
Both comments addressed within https://github.com/handsontable/handsontable/pull/8897/commits/dc218b6ca45036539140e1a8bfe6604256544b60.

@kirszenbaum Please check how it looks like 🙂